WordleGame/WordleGame/Models/Game/GameManager.swift

```diff
@@ -141,10 +157,24 @@ struct GameManager {
     // MARK: Handle game end
     
     private func handleWin() {
-        print("Win")
+        self.resultDelegate?.saveGameScore(score: scorePoints)
+        self.delegate?.handleWin(numberOfAttempts: currentAttemptIndex + 1)
     }
     
     private func handleLose() {
-        print("Lose")
+        self.delegate?.handleLose()
+    }
+    
+    // MARK: Save data
+    
+    func saveData(userName: String) {
+        let gameResult = GameResult(userName: userName,
+                                    score: scorePoints,
+                                    time: gameTime)
+        
+        UserDefaultsService.shared.saveGameResult(gameResult)
+        
+        let gameResults = UserDefaultsService.shared.getGameResults()
+        print(gameResults)
```

Comment:
Do you still need this code?

Reply:
Nope.) I used this code to get information from code and see it.)